compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -32,6 +34,7 @@
 namespace mlir::iree_compiler::IREE::GPU {
 
 constexpr int64_t kCacheLineSizeBits = 128 * 8;
+static constexpr int64_t kPreferredCopyNumBits = 128;
```

Comment:
nit: constexpr in the global scope implies static, so this is redundant
```suggestion
constexpr int64_t kPreferredCopyNumBits = 128;
```